tests/nlu/test_evaluation.py

```diff
@@ -410,17 +408,11 @@ def test_run_cv_evaluation(
             "pipeline": [
                 {"name": "WhitespaceTokenizer"},
                 {"name": "CountVectorsFeaturizer"},
-                {"name": "DIETClassifier", EPOCHS: 2},
+                {"name": "DIETClassifier", EPOCHS: 25},
```

Comment:
can we instead set the random seed?

Reply:
potentially, but the difference here is really just 5 seconds or so. The training data is extremely small